src/runtime_src/core/edge/drm/zocl/zocl_xclbin.c

```diff
@@ -1166,7 +1174,21 @@ zocl_load_sect(struct drm_zocl_dev *zdev, struct axlf *axlf, char __user *xclbin
 static bool
 is_aie_only(struct axlf *axlf)
 {
-	return (axlf->m_header.m_actionMask & AM_LOAD_AIE);
+	if (!xrt_xclbin_get_section_num(axlf, IP_LAYOUT) &&
```

Comment:
have the old check also, axlf->m_header.m_actionMask & AM_LOAD_AIE OR new condition